libc/test/UnitTest/LibcTest.cpp

```diff
@@ -147,7 +140,7 @@ int Test::runTests(const char *TestFilter) {
       break;
     case RunContext::RunResult::Pass:
       tlog << GREEN << "[       OK ] " << RESET << TestName;
-#if __STDC_HOSTED__ || defined(LIBC_TARGET_ARCH_IS_GPU)
+#ifdef LIBC_TEST_USE_CLOCK
```

Comment:
This change looks fine to me, though in future I think it would be useful to have it report the time regardless of if the test passes or fails.

Reply:
This guards usage of the `clock()` function. We could potentially make it print something if unimplemented, or default to just return zero or something.